src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/AddAzureRmDataLakeAnalyticsDataSource.cs

```diff
@@ -53,14 +53,6 @@ public class AddAzureDataLakeAnalyticsDataSource : DataLakeAnalyticsCmdletBase
         [ValidateNotNullOrEmpty]
         public string AccessKey { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = false,
-            ParameterSetName = DataLakeParameterSetName,
-            HelpMessage =
-                "Optionally indicates that this should now be the default storage account for the DataLakeAnalytics account."
-            )]
-        [ValidateNotNullOrEmpty]
-        public SwitchParameter Default { get; set; }
```

Comment:
Because it was a bug. If any customer tried to pass this parameter the cmdlet would fail (always) since this has never been supported. It was a test gap miss that was corrected recently during a test debt payment.